src/urllib3/connection.py

```diff
@@ -108,21 +104,10 @@ def __init__(self, *args, **kw):
         if six.PY3:  # Python 3
             kw.pop('strict', None)
 
-        # Pre-set source_address in case we have an older Python like 2.6.
-        self.source_address = kw.get('source_address')
```

Comment:
I think we should keep this attribute on the `HTTPConnection` object, might be being used by others.

Reply:
Kept.